packages/itinerary-body/src/place-row.js

```diff
@@ -107,7 +104,10 @@ const PlaceRow = ({
                 TransitLegSummary={TransitLegSummary}
                 transitOperator={
                   leg.agencyId &&
-                  getTransitOperatorFromConfig(leg.agencyId, config)
+                  coreUtils.route.getTransitOperatorFromId(
+                    leg.agencyId,
```

Comment:
I think this should be using `coreUtils.route.getTransitOperatorFromOtpRoute(leg.routeId, config.transitOperators)`. The input arg for `getTransitOperatorFromId` is `feedId` not `agencyId`. This is an undocumented bug in this component. Related bug that could be addressed in this PR: https://github.com/opentripplanner/otp-ui/issues/190